mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,20 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = (req, res) => { 
```

Comment:
The function name `exports.removeOrder` in the controller does not match the function name `remove` in the router. This inconsistency can lead to errors and confusion. It should be renamed to `exports.remove` to match the route function name.